project/cmake/scripts/linux/Install.cmake

```diff
@@ -38,98 +38,273 @@ configure_file(${CORE_SOURCE_DIR}/tools/Linux/kodi-standalone.sh.in
 # Configure cmake files
 configure_file(${PROJECT_SOURCE_DIR}/KodiConfig.cmake.in
                ${CORE_BUILD_DIR}/scripts/${APP_NAME}Config.cmake @ONLY)
+configure_file(${PROJECT_SOURCE_DIR}/XBMCConfig.cmake.in
+               ${CORE_BUILD_DIR}/scripts/XBMCConfig.cmake @ONLY)
```

Comment:
This file was kept for backwards compatibility. Somebody then went ahead and renamed it so it serves no purpose. Nuke or bring back old naming (only one with much legacy is me and i have updated all my stuff so i vote nuke.